### ISSUE TITLE
I am Getting an iopaint not installed issue

### ISSUE THREAD

**Issue by shivmsha776-dot** [D-Ogi/WatermarkRemover-AI#62]
This is what its showing, and iopaint is not installing 

  =============================================
     WatermarkRemover-AI Setup
  =============================================

  [?] Are you in China? (y/n)
      This will use faster mirrors for downloads
      : n
  [OK] Using default mirrors

  [OK] Python found

  [*] Installing dependencies...
      This takes 5-10 minutes. Chill and learn something!

      Did you know?

Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu124
Requirement already up-to-date: torch>=2.4.0 in .\python\Lib\site-packages (from -r requirements.txt (line 3)) (2.10.0)
Requirement already up-to-date: torchvision>=0.19.0 in .\python\Lib\site-packages (from -r requirements.txt (line 4)) (0.25.0)
Requirement already up-to-date: transformers>=4.50.0 in .\python\Lib\site-packages (from -r requirements.txt (line 5)) (5.0.0)
      [i] Florence-2 can detect watermarks in any language - even emojis!                 Requirement already up-to-date: diffusers>=0.30.0 in .\python\Lib\site-packages (from -r requirements.txt (line 6)) (0.36.0)
      [i] The AI model was trained on millions of images to understand context            Requirement already up-to-date: numpy<2 in .\python\Lib\site-packages (from -r requirements.txt (line 7)) (1.26.4)
Requirement already up-to-date: opencv-python-headless<4.12.0,>=4.8.0 in .\python\Lib\site-packages (from -r requirements.txt (line 10)) (4.11.0.86)
      [i] LaMA stands for 'Large Mask inpainting' - it fills gaps naturally               Requirement already up-to-date: Pillow>=10.0.0 in .\python\Lib\site-packages (from -r requirements.txt (line 11)) (12.1.0)
Requirement already up-to-date: pywebview>=4.0 in .\python\Lib\site-packages (from -r requirements.txt (line 16)) (6.1)
Requirement already up-to-date: loguru in .\python\Lib\site-packages (from -r requirements.txt (line 19)) (0.7.3)
Requirement already up-to-date: click in .\python\Lib\site-packages (from -r requirements.txt (line 20)) (8.3.1)
Requirement already up-to-date: tqdm in .\python\Lib\site-packages (from -r requirements.txt (line 21)) (4.67.3)
      [?] Tip: GPU processing is 10-50x faster than CPU                                   Requirement already up-to-date: psutil in .\python\Lib\site-packages (from -r requirements.txt (line 22)) (7.2.2)
Requirement already up-to-date: pyyaml in .\python\Lib\site-packages (from -r requirements.txt (line 23)) (6.0.3)
Requirement already satisfied, skipping upgrade: filelock in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (3.20.3)
Requirement already satisfied, skipping upgrade: typing-extensions>=4.10.0 in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (4.15.0)
Requirement already satisfied, skipping upgrade: sympy>=1.13.3 in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (1.14.0)
Requirement already satisfied, skipping upgrade: networkx>=2.5.1 in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (3.6.1)
Requirement already satisfied, skipping upgrade: jinja2 in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (3.1.6)
Requirement already satisfied, skipping upgrade: fsspec>=0.8.5 in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (2026.1.0)
Requirement already satisfied, skipping upgrade: setuptools; python_version >= "3.12" in .\python\Lib\site-packages (from torch>=2.4.0->-r requirements.txt (line 3)) (80.10.2)
Requirement already satisfied, skipping upgrade: huggingface-hub<2.0,>=1.3.0 in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (1.3.7)
Requirement already satisfied, skipping upgrade: packaging>=20.0 in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (26.0)
Requirement already satisfied, skipping upgrade: regex!=2019.12.17 in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (2026.1.15)
Requirement already satisfied, skipping upgrade: tokenizers<=0.23.0,>=0.22.0 in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (0.22.2)
Requirement already satisfied, skipping upgrade: typer-slim in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (0.21.1)
Requirement already satisfied, skipping upgrade: safetensors>=0.4.3 in .\python\Lib\site-packages (from transformers>=4.50.0->-r requirements.txt (line 5)) (0.7.0)
Requirement already satisfied, skipping upgrade: importlib_metadata in .\python\Lib\site-packages (from diffusers>=0.30.0->-r requirements.txt (line 6)) (8.7.1)
Requirement already satisfied, skipping upgrade: httpx<1.0.0 in .\python\Lib\site-packages (from diffusers>=0.30.0->-r requirements.txt (line 6)) (0.28.1)
Requirement already satisfied, skipping upgrade: requests in .\python\Lib\site-packages (from diffusers>=0.30.0->-r requirements.txt (line 6)) (2.32.5)
Requirement already satisfied, skipping upgrade: pythonnet; sys_platform == "win32" in .\python\Lib\site-packages (from pywebview>=4.0->-r requirements.txt (line 16)) (3.0.5)
Requirement already satisfied, skipping upgrade: proxy_tools in .\python\Lib\site-packages (from pywebview>=4.0->-r requirements.txt (line 16)) (0.1.0)
Requirement already satisfied, skipping upgrade: bottle in .\python\Lib\site-packages (from pywebview>=4.0->-r requirements.txt (line 16)) (0.13.4)
Requirement already satisfied, skipping upgrade: colorama>=0.3.4; sys_platform == "win32" in .\python\Lib\site-packages (from loguru->-r requirements.txt (line 19)) (0.4.6)
Requirement already satisfied, skipping upgrade: win32-setctime>=1.0.0; sys_platform == "win32" in .\python\Lib\site-packages (from loguru->-r requirements.txt (line 19)) (1.2.0)
Requirement already satisfied, skipping upgrade: mpmath<1.4,>=1.1.0 in .\python\Lib\site-packages (from sympy>=1.13.3->torch>=2.4.0->-r requirements.txt (line 3)) (1.3.0)
Requirement already satisfied, skipping upgrade: MarkupSafe>=2.0 in .\python\Lib\site-packages (from jinja2->torch>=2.4.0->-r requirements.txt (line 3)) (3.0.3)
Requirement already satisfied, skipping upgrade: hf-xet<2.0.0,>=1.2.0; platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "arm64" or platform_machine == "aarch64" in .\python\Lib\site-packages (from huggingface-hub<2.0,>=1.3.0->transformers>=4.50.0->-r requirements.txt (line 5)) (1.2.0)
Requirement already satisfied, skipping upgrade: shellingham in .\python\Lib\site-packages (from huggingface-hub<2.0,>=1.3.0->transformers>=4.50.0->-r requirements.txt (line 5)) (1.5.4)
Requirement already satisfied, skipping upgrade: zipp>=3.20 in .\python\Lib\site-packages (from importlib_metadata->diffusers>=0.30.0->-r requirements.txt (line 6)) (3.23.0)
Requirement already satisfied, skipping upgrade: anyio in .\python\Lib\site-packages (from httpx<1.0.0->diffusers>=0.30.0->-r requirements.txt (line 6)) (4.12.1)
Requirement already satisfied, skipping upgrade: certifi in .\python\Lib\site-packages (from httpx<1.0.0->diffusers>=0.30.0->-r requirements.txt (line 6)) (2026.1.4)
Requirement already satisfied, skipping upgrade: httpcore==1.* in .\python\Lib\site-packages (from httpx<1.0.0->diffusers>=0.30.0->-r requirements.txt (line 6)) (1.0.9)
Requirement already satisfied, skipping upgrade: idna in .\python\Lib\site-packages (from httpx<1.0.0->diffusers>=0.30.0->-r requirements.txt (line 6)) (3.11)
Requirement already satisfied, skipping upgrade: charset_normalizer<4,>=2 in .\python\Lib\site-packages (from requests->diffusers>=0.30.0->-r requirements.txt (line 6)) (3.4.4)
Requirement already satisfied, skipping upgrade: urllib3<3,>=1.21.1 in .\python\Lib\site-packages (from requests->diffusers>=0.30.0->-r requirements.txt (line 6)) (2.6.3)
Requirement already satisfied, skipping upgrade: clr_loader<0.3.0,>=0.2.7 in .\python\Lib\site-packages (from pythonnet; sys_platform == "win32"->pywebview>=4.0->-r requirements.txt (line 16)) (0.2.10)
Requirement already satisfied, skipping upgrade: h11>=0.16 in .\python\Lib\site-packages (from httpcore==1.*->httpx<1.0.0->diffusers>=0.30.0->-r requirements.txt (line 6)) (0.16.0)
Requirement already satisfied, skipping upgrade: cffi>=1.17; python_version >= "3.8" in .\python\Lib\site-packages (from clr_loader<0.3.0,>=0.2.7->pythonnet; sys_platform == "win32"->pywebview>=4.0->-r requirements.txt (line 16)) (2.0.0)
Requirement already satisfied, skipping upgrade: pycparser; implementation_name != "PyPy" in .\python\Lib\site-packages (from cffi>=1.17; python_version >= "3.8"->clr_loader<0.3.0,>=0.2.7->pythonnet; sys_platform == "win32"->pywebview>=4.0->-r requirements.txt (line 16)) (3.0)

  [*] Installing iopaint (no deps)...
Requirement already satisfied: iopaint in .\python\Lib\site-packages (1.6.0)

  [X] Failed to install iopaint
  Press Enter to exit:

**Comment by D-Ogi** [comment]
Duplicate of #49: pip reports IOPaint installed/already installed, then setup.ps1 reports failure. The later missing-dependency errors may follow from the installer stopping before its next step; they do not prove that the IOPaint package itself was never installed. Further installer-status investigation is tracked in #49, with dependency compatibility in #48.
